packages/frontend/src/components/common/ResourceInfoPopup/DashboardList.tsx

```diff
@@ -0,0 +1,37 @@
+import { Anchor, List, Text } from '@mantine/core';
+import { FC } from 'react';
+import { useDashboardsContainingChart } from '../../../hooks/dashboard/useDashboards';
+
+type Props = {
+    resourceItemId: string;
+    projectUuid: string;
+};
+
+export const DashboardList: FC<Props> = ({ resourceItemId, projectUuid }) => {
+    const { data: relatedDashboards } = useDashboardsContainingChart(
+        projectUuid,
+        resourceItemId,
+    );
+    return (
+        <>
+            <Text fw={600} fz="xs" color="gray.6">
+                Used in {relatedDashboards?.length ?? 0} dashboard
+                {relatedDashboards?.length === 1 ? '' : 's'}
+                {relatedDashboards && relatedDashboards.length > 0 ? ':' : ''}
+            </Text>
+            {!!relatedDashboards?.length && (
+                <List size="xs">
+                    {relatedDashboards.map(({ uuid, name }) => (
+                        <List.Item key={uuid}>
+                            <Anchor
+                                href={`/projects/${projectUuid}/dashboards/${uuid}/view`}
```

Comment:
or perhaps use a style minimal button that does a  `history.push` onClick where we can add stopPropagation and preventDefault to fix the issue I previously mentioned

Reply:
OK, happy to do that. I didn't change this -- is the Link a new pattern for us?

Reply:
@rephus history.push is a bad practice because it does not act as a link. and when you mod+click it does not open in a new tab. Mantine polymorphism `<Anchor component={Link} to={` solves both of the problems